### PR TITLE
Add ru.json

### PR DESCRIPTION
**Pull request by Dunzee**
d4nzee#1337

### CODE REVIEW

File: ru.json [hunk 1]
```diff
@@ -0,0 +1,274 @@
+{
+    "name" : "Russian",
```
Comment (njb-said) [inline ru.json]: Could you change this to the name of the language in Russian, and add a `name-en` field underneath as with the name in english:

`"name-en" : "Russian"`

File: ru.json [hunk 2]
```diff
@@ -0,0 +1,275 @@
+{
+    "name" : "Русский",
+    "name-en" : "Russian",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : true,
```
Comment (Krymonota) [inline ru.json]: Russian doesn't use "th" for dates. Please set this from `true` to `false`.

File: ru.json [hunk 3]
```diff
@@ -0,0 +1,275 @@
+{
+    "name" : "Русский",
+    "name-en" : "Russian",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : true,
+            "dateFormat" : "[month] [date] [year]",
```
Comment (Krymonota) [inline ru.json]: The correct date format for Russian is `[date] [month] [year]`.

File: ru.json [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+            "title-all" : "Все предметы",
+            "error" : "Невозможно загрузить предметы, попробуйте позже.",
+            "error-too-long" : "Это займет больше, чем ожидалось"
+        },
+        "shop" : {
+            "title" : "Сегодняшний магазин",
+            "title-featured" : "Рекомендуемые",
+            "title-daily" : "Ежедневный",
+            "error" : "Магазин недоступен, попробуйте позже",
+            "line1" : "Сегодняшний магазин предметов в Fortnite Battle Royale - обновляется каждый день в [00:00 UTC](https://time.is/UTC|tooltip=Current time in UTC|newtab=true).",
```
Comment (Krymonota) [inline ru.json]: Please translate the tooltip. My Russian is a bit rusty, but I think you can replace "Current time in UTC" with "Текущее время в UTC".

File: ru.json [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+            "line2" : "Вы можете посмотреть вчерашний магазин [здесь](/shop/yesterday|tooltip=Yesterday's Item Shop).",
```
Comment (Krymonota) [inline ru.json]: Please translate the tooltip. "Yesterday's Item Shop" could be replaced with "Вчерашний магазин".

**Comment by Krymonota** [comment]
Looks good now! 
However, would it be possible for you also to translate the newly added voting strings? That would be amazing! 

https://github.com/fnbrco/translations/blob/e439ce7d7af235a55feb36c8ac796ea5ecb90d64/en.json#L114-L134

**Pull request merged**